docs/docs/queries/structure.md

```diff
@@ -139,15 +139,16 @@ SORT file.ctime DESC
 LIMIT 10
 ```
 
-Lists files that have the metadata field `contacts` and where `contacts` is a list (a multi value), as well as contain the page `Mr. L`. Sort after the length (count) of contacts and sort the contacts itself after the contact's age in ascending order.
+Lists the 10 oldest and incompleted tasks of your vault as an interactive task list, grouped by their containing file and sorted from oldest to newest affected file.
```

Comment:
Lists the 10 oldest, incomplete tasks [...] sorted from oldest creation time to newest.

Not sure what affected file means here.